partitura/tokenizer/remi.py

```diff
@@ -0,0 +1,493 @@
+from fractions import Fraction
+from math import lcm, isqrt
+from collections import defaultdict
+import numpy as np
+
+from ..utils.globals import REGULAR_NUM_DENOM, VALID_TIME_SIGNATURES, PROGRAM_INSTRUMENT_MAP
+from ..score import Part, Note, TimePoint
+from .utils import midi_to_step_octave_alter
+
+class REMITokenizer:
+    """
+    REMI-style Tokenizer for symbolic music encoding.
+
+    Attributes:
+        config (dict): Configuration dictionary for tokenizer options.
+        vocabulary (dict): Mapping from event string to index.
+        inverse_vocabulary (dict): Mapping from index to event string.
+        events (list): List of tokenized events in string format.
+        tune_in_index (list): List of tokenized events in index format.
+    """
```

Comment:
Although not necessary, I think a more detailed description in the doc line would be nice, including the reference for REMI and an example use case under the `Example:` doc line.